src/Packetery/Module/Carrier/OptionsPage.php

```diff
@@ -101,25 +102,25 @@ class OptionsPage {
 	private $carDeliveryConfig;
 
 	/**
-	 * WC Native carrier settings config.
+	 * Options provider.
 	 *
-	 * @var WcSettingsConfig
+	 * @var Provider
 	 */
-	private $wcSettingsConfig;
+	private $optionsProvider;
 
 	/**
 	 * OptionsPage constructor.
 	 *
-	 * @param Engine                    $latteEngine        PacketeryLatte_engine.
-	 * @param EntityRepository          $carrierRepository  Carrier repository.
-	 * @param FormFactory               $formFactory        Form factory.
-	 * @param Request                   $httpRequest        Packetery\Nette Request.
-	 * @param CountryListingPage        $countryListingPage CountryListingPage.
-	 * @param MessageManager            $messageManager     Message manager.
-	 * @param PacketaPickupPointsConfig $pickupPointsConfig Internal pickup points config.
-	 * @param FeatureFlagManager        $featureFlag        Feature flag.
-	 * @param CarDeliveryConfig         $carDeliveryConfig  Car delivery config.
-	 * @param WcSettingsConfig          $wcSettingsConfig   WC Native carrier settings config.
+	 * @param Engine                    $latteEngine                   PacketeryLatte_engine.
```

Comment:
Proč to odsazení popisů?

Reply:
Vypada to, jako ze jsem udelal nejak divne ten rebase. Vratil jsem to do puvodniho stavu.